docs/ota-client-guide/modules/ROOT/pages/recommended-clientconfig.adoc

```diff
@@ -1,79 +1,89 @@
 = Recommended client configurations
 
-Before you start developing or deploying to production, you should check that your configuration file has appropriate settings for your use case. 
+Before you start developing or deploying to production, you should check that your configuration file has appropriate settings for your use case.
 
-* If you are running standalone aktualizr, your configuration file will be a **.toml** file.
-+
-You configure aktualizr by creating and updating a `*.toml` file in one of the following directories:
+The configuration file and exact parameters depend on how you use OTA Connect:
 
-** `/usr/lib/sota/conf.d`
-** `/etc/sota/conf.d/`
+* If you're building disk images that contain aktualizr, you need to update your *build configuration*.
+* If you're using a standalone version of aktualizr that you've installed locally, you need to update the *client configuration*.
+ 
```

Comment:
what if I am building disk images that contain libaktualizr ?

Reply:
I don't know. This PR is more about changing the layout of the existing content. @patrickvacek do the build config parameters work the same way if you add a libaktualizr integration to your build (like this `IMAGE_INSTALL_append = " libaktualizr-demo"`).
For example would the polling interval config work in the same way as building with normal aktualizr?

Reply:
> do the build config parameters work the same way if you add a libaktualizr integration to your build

That's a good question, I haven't tried it myself. AFAIK, yes, it should. The config fragments in meta-updater do not depend on the aktualizr recipe. However, you can use libaktualizr-demo-app without using meta-updater, so that's up to the user to decide. @eu-smirnov can you confirm my understanding?